src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -55,17 +55,17 @@ const static int LEGACY_COUNT = 4;
 static int
 get_render_value(const std::string dir)
 {
-  static const std::string render_name = "renderD"; 
+  static const std::string render_name = "renderD";
   int instance_num = INVALID_ID;
   std::string sub;
 
   boost::filesystem::path render_dirs(dir);
   if (!boost::filesystem::is_directory(render_dirs))
     return instance_num;
- 
+
   boost::filesystem::recursive_directory_iterator end_iter;
     for(boost::filesystem::recursive_directory_iterator iter(render_dirs); iter != end_iter; ++iter) {
-      if (iter->path().filename().string().compare(0,render_name.size(),render_name)== 0) {   
+      if (iter->path().filename().string().compare(0,render_name.size(),render_name)== 0) {
```

Comment:
While you are here, contribute to a better world by using `==` instead of compare. :-)

Reply:
> While you are here, contribute to a better world by using `==` instead of `compare`. :-)
haha, sure, didn't see that :-)